### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**
Screenshot 2022-10-31 154903
Screenshot 2022-10-31 154407
Screenshot 2022-10-31 154711
Screenshot 2022-10-31 154733
Screenshot 2022-10-31 154806
Screenshot 2022-10-31 154835
Screenshot 2022-10-31 155334

> please, add a screenshot

Screenshot 2022-11-07 202556

### CODE REVIEW

File: taxi/views.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render
+from django.http import HttpResponseRedirect
```
Comment (mltdll) [inline taxi/views.py:2]: This import doesn't seem to be used.

File: taxi/forms.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta:
+        model = Driver
```
Comment (mltdll) [inline taxi/forms.py:11]: It's probably better to use `get_user_model()` here as well, even though this class is strongly dependent on **this** particular user model. But maybe it's even better to ask mentors.

**Review by mltdll — CHANGES_REQUESTED** [review]

**Review by mltdll — APPROVED** [review]

**Comment by Nattalli** [comment]
add a screenshot for this: `On car creation page switch to checkboxes widget for assigning drivers to car.`

**Review by Nattalli — CHANGES_REQUESTED** [review]
please, add a screenshot

**Review by Nattalli — APPROVED** [review]

**Pull request closed without merging**